README.md

```diff
@@ -94,73 +94,80 @@ The plugin comes with the following defaults:
     },
   },
 
-  -- file-type specific options
-  -- any options that are passed here will override the default config
+  -- file type specific opts
```

Comment:
I think it's better to write options than opts here?

Also, I think it's fine to keep the dash; "file-type specific" seems correct to me.

Reply:
Yes, options are better! I had a variable called `options`, but renamed it to `opts` to better align with the lazy.nvim `opts` table. Seems like it also affected the comments. Reverted now. 

Seems to me like the Neovim docs mostly uses "filetype", in one word, although "file type" is the grammatically correct version. I will stick with "filetype" in this case, since this is a Neovim plugin after all.